src/parser/classes/Command.ts

```diff
@@ -0,0 +1,30 @@
+import {YTNode} from '../helpers.js';
+import type { RawNode } from '../index.js';
+import NavigationEndpoint from './NavigationEndpoint.js';
+
+export default class Command extends YTNode {
+  static type = 'Command';
+
+  click_tracking_params: string;
+  command_metadata: {
+    web_command_metadata: {
+      url: string,
+      web_page_type: string,
+      root_ve: number
+    }
+  };
+  endpoint: NavigationEndpoint;
+
+  constructor(data: RawNode) {
+    super();
+    this.click_tracking_params = data.clickTrackingParams;
```

Comment:
You don't need to save or extract YouTube's tracking params.

Reply:
Removed